central/cluster/datastore/datastore.go

```diff
@@ -293,5 +297,5 @@ func GetTestRocksBleveDataStore(t *testing.T, rocksengine *rocksdbBase.RocksDB,
 		alertStore, namespaceStore, deploymentStore,
 		nodeStore, podStore, secretStore, netFlowStore, netEntityStore,
 		serviceAccountStore, k8sRoleStore, k8sRoleBindingStore, sensorCnxMgr, nil,
-		dacky, clusterRanker, networkBaselineManager)
+		dacky, clusterRanker, networkBaselineManager, nil)
```

Comment:
`nil` because this is only postgres path. The non-postgres path is handled thru cluster cve manager.